src/hotspot/cpu/s390/macroAssembler_s390.cpp

```diff
@@ -3251,27 +3259,40 @@ void MacroAssembler::compiler_fast_unlock_object(Register oop, Register box, Reg
 
   // Handle existing monitor.
   // The object has an existing monitor iff (mark & monitor_value) != 0.
-  z_lg(currentHeader, oopDesc::mark_offset_in_bytes(), oop);
+  z_lg(currentHeader, hdr_offset, oop);
   guarantee(Immediate::is_uimm16(markWord::monitor_value), "must be half-word");
+  if (LockingMode == LM_LIGHTWEIGHT)
+    z_lgr(temp, currentHeader);
```

Comment:
Hotspot coding style: Use curly brackets even if it's only one statement.

Reply:
Thanks for comments, 

Fixed.